src/components/card/card.tsx

```diff
@@ -33,17 +33,32 @@ const layoutToClassNameMap: { [layout in CardLayout]: string } = {
 
 export const LAYOUT_ALIGNMENTS = keysOf(layoutToClassNameMap);
 
+type CardDisplay = 'panel' | 'plain';
+
+const displayToClassNameMap: { [display in CardDisplay]: string } = {
+  panel: '',
+  plain: 'euiCard--plain',
+};
+
+export const DISPLAYS = keysOf(displayToClassNameMap);
+
 type EuiCardProps = Omit<CommonProps, 'aria-label'> & {
   /**
    * Card's are required to have at least a title and description
    */
   title: NonNullable<ReactNode>;
 
   /**
-   * Determines the title's heading element.
+   * Determines the title's heading element
    */
   titleElement?: 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'span';
 
+  /**
+   * Determines the title's size, matching that of EuiTitle.
+   * Though, card titles can't be too large or smaller than the description text.
```

Comment:
Maybe not getting to the literal point (can't have title smaller than desc), but might be a bit easier to read:
```suggestion
   * Though, card titles can't be too large or small relative to the description text.
```